LDAR_Sim/testing/unit_testing/test_programs/test_method/test_deploy_crew_costs.py

```diff
@@ -0,0 +1,64 @@
+"""
+------------------------------------------------------------------------------
+Program:     The LDAR Simulator (LDAR-Sim)
+File:        test_deploy_crew_costs.py
+Purpose: Unit test for testing the cost portion of crew deployment
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the MIT License as published
+by the Free Software Foundation, version 3.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+MIT License for more details.
+You should have received a copy of the MIT License
+along with this program.  If not, see <https://opensource.org/licenses/MIT>.
+
+------------------------------------------------------------------------------
+"""
+
+from file_processing.output_processing.output_utils import CrewDeploymentStats
+from src.programs.method import Method
+from testing.unit_testing.test_programs.test_method.method_testing_fixtures import (  # noqa
+    deploy_crews_testing3_fix,
+    deploy_crews_testing4_fix,
+    deploy_crews_testing5_fix,
+)
+from src.scheduling.workplan import Workplan
+
+
+def test_000_simple_single_crew_daily_cost(deploy_crews_testing3):
```

Comment:
Under the test definitions can you briefly describe key inputs, expected outputs and the point of the test for each test, given that the test data is obfuscated by fixtures.